src/modules/tests/render.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const getcom = require('../render.js');
```

Comment:
I see that you install Babel plugin in your project, in order to automate the work of transform all ES6 modules into CommonJS notation,  I think it would be better if you use import instead of require.😊 You can learn more in this [link] (https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/lesson_more_about_unit_tests_in_js.md)